src/TokenAirdrop.hs

```diff
@@ -31,7 +32,7 @@ tokenAirdrop config = do
             map
               ( \beneficiary ->
                   let val = Value.assetClassValue beneficiary.assetClass beneficiary.amount
-                   in (Constraints.mustPayToPubKey beneficiary.address.pkaPubKeyHash val, [beneficiary])
```

Comment:
We have an opportunity to fix an annoying hack here!
Lets not tackle it here, I've made a ticket here though #25